<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>mfsk decoder</string>
    </property>
   </widget>
   <widget class="QLabel" name="mfsktextBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mfskSquelch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QComboBox" name="mfskMode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>mfsk 16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mfsk 8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mfsk 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mfsk 11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mfsk 22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mfsk 31</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mfsk 32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mfsk 64</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="mfskReverse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>reverse</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="mfskAfcon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>afc off</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>afc on</string>
     </property>
    </item>
   </widget>
   <widget class="QLCDNumber" name="s2nDisplay">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>12</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="baudrateEstimate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="cfDisplay">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
